Sources/Spawn/Spawn.swift

```diff
@@ -83,16 +84,42 @@ public final class Spawn {
             dynamicBuffer.deallocate(capacity: bufferSize)
             return nil
         }
+        #else
+        func callback(x: UnsafeMutableRawPointer?) -> UnsafeMutableRawPointer? {
```

Comment:
I know that this repo is not mine or anything; but as seeing that the code inside the `Linux` path doesn't contain stuff that would fail compiling on `macOS`; wouldn't it be better to have something like `macOSCallback` and `linuxCallback` and then only applying the filtering inside of `callback`:
```swift
func callback(x: UnsafeMutableRawPointer?) -> UnsafeMutableRawPointer? {
#if(OSX)
   return macOSCallback(x)
#else
   return linuxCallback(x)
#endif
}
```